app/src/src/screens/menu/ProfileScreen.jsx

```diff
@@ -34,32 +35,46 @@ class ProfileScreen extends Component {
     }
   };
 
+  componentWillUnmount() {
+    this.setState = (_state, _callback) => {
+      return;
+    };
+  }
+
   render() {
     return (
       <View style={styles.container}>
         <Header title="Profile" />
-        <View style={styles.backHeading}>
-          <Image
-            style={styles.backGroundHeading}
-            source={{ uri: this.state.user.background_photo }}
-          />
-          <View style={styles.profileArea}>
-            <TouchableOpacity
-              style={styles.profileImg}
-              onPress={() => this.props.navigation.navigate(routes.menuHome)}
-            >
-              <Avatar.Image
-                size={100}
-                style={styles.pfpShadow}
-                source={{ uri: this.state.user.profile_photo }}
-              />
-            </TouchableOpacity>
-          </View>
-        </View>
         <ScrollView
-          style={{ flex: 1, marginHorizontal: 1 }}
-          scrollEnabled={Platform.OS === 'android'}
+          style={styles.scrollViewStyle}
+          contentContainerStyle={styles.contentContainer}
         >
+          <View style={styles.backHeading}>
+            <Image
+              style={styles.backGroundHeading}
+              source={{
+                uri: this.state.user.background_photo
+                  ? this.state.user.background_photo
+                  : '//:0',
```

Comment:
This can be simplified to 
`this.state.user.background_photo ?? '//:0'`
This is called 'nullish coalescing', where the value on the right side of the ?? is used only if the left side is null or undefined. Otherwise, it will use the value on the left side.